sycl/include/CL/sycl/builtins.hpp

```diff
@@ -1102,198 +1102,139 @@ fast_normalize(T p) __NOEXC {
 detail::common_rel_ret_t<T> isfinite(T x) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_IsFinite<detail::rel_ret_t<T>>(x));
 }
 
-// int isinf (half x)
-// shortn isinf (halfn x)
-// igeninteger32bit isinf (genfloatf x)
-// int isinf (double x)
-// longn isinf (doublen x)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> isinf(T x) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_IsInf<detail::rel_ret_t<T>>(x));
 }
 
-// int isnan (half x)
-// shortn isnan (halfn x)
-// igeninteger32bit isnan (genfloatf x)
-// int isnan (double x)
-// longn isnan (doublen x)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> isnan(T x) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_IsNan<detail::rel_ret_t<T>>(x));
 }
 
-// int isnormal (half x)
-// shortn isnormal (halfn x)
-// igeninteger32bit isnormal (genfloatf x)
-// int isnormal (double x)
-// longn isnormal (doublen x)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> isnormal(T x) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_IsNormal<detail::rel_ret_t<T>>(x));
 }
 
-// int isordered (half x)
-// shortn isordered (halfn x, halfn y)
-// igeninteger32bit isordered (genfloatf x, genfloatf y)
-// int isordered (double x, double y)
-// longn isordered (doublen x, doublen y)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> isordered(T x, T y) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_Ordered<detail::rel_ret_t<T>>(x, y));
 }
 
-// int isunordered (half x, half y)
-// shortn isunordered (halfn x, halfn y)
-// igeninteger32bit isunordered (genfloatf x, genfloatf y)
-// int isunordered (double x, double y)
-// longn isunordered (doublen x, doublen y)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> isunordered(T x, T y) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_Unordered<detail::rel_ret_t<T>>(x, y));
 }
 
-// int signbit (half x)
-// shortn signbit (halfn x)
-// igeninteger32bit signbit (genfloatf x)
-// int signbit (double)
-// longn signbit (doublen x)
 template <typename T,
           typename = detail::enable_if_t<detail::is_genfloat<T>::value, T>>
 detail::common_rel_ret_t<T> signbit(T x) __NOEXC {
   return detail::RelConverter<T>::apply(
       __sycl_std::__invoke_SignBitSet<detail::rel_ret_t<T>>(x));
 }
 
-// int any (sigeninteger x)
+// The standard is being corrected for "any" at
+// https://github.com/KhronosGroup/SYCL-Docs/pull/234. Scalar/marray version
```

Comment:
That PR hasn't landed yet, so why it is OK to assume it will land and and in the way it is spelled currently?
@gmlueck : could you also confirm that this implementation change is valid now?

Reply:
We discussed this in the Khronos SYCL workgroup, and there was general agreement on the direction.  I cannot say for sure that it will be accepted until it is merged, though.  I do not mind if we put this PR on hold until  the spec changes is accepted.  There is a Khronos meeting today, but our next regular "issue" meeting is on April 14.  There is some chance the spec PR will get merged today, but most likely it won't happen until April 14.

Reply:
I did review and it looks good to me.

Reply:
Note that the Khronos spec PR (https://github.com/KhronosGroup/SYCL-Docs/pull/234) was accepted by the committee and merged.